.editorconfig

```diff
@@ -0,0 +1,33 @@
+
+################################################################################
+#                                                                              #
+#                             Ｅｄｉｔｏｒｃｏｎｆｉｇ                             #
+#                                                                              #
+#                 This file defines basic file properties that                 #
+#                 editors / IDEs use when you modify documents.                #
+#                                                                              #
+#                 Check https://EditorConfig.org for details.                  #
+#                                                                              #
+################################################################################
+
+
+root = true
+
+
+[*]
+
+end_of_line = lf
+
+[LICENSE]
+[*.{editorconfig,gitignore,json,ts,md}]
+
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+charset = utf-8
+
+
+[*.md]
+
+trim_trailing_whitespace = false
```

Comment:
What about this instead? I think the comment at top is unnecessary and the ident size should be 2.

```
root = true

[*]
end_of_line = lf
insert_final_newline = true
indent_style = space
indent_size = 2
charset = utf-8
trim_trailing_whitespace = true
```